src/components/home/elements/TokenRecords.vue

```diff
@@ -34,6 +37,9 @@
        class="records">
     <div class="title-records">
       <span>Token Watch List</span>
+      <b-button class="add-token-button text-decoration-none"
+                variant="link"
+                v-b-modal.addTokenModal>&oplus;</b-button>
```

Comment:
Line 2 - 35 is almost the same as Line 36 - 54. Refactor this part to simplify the code.